specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 1980:
```
  national_contest_table: TYPE =
    sequence[[nation, contest]]
  state_contest_table: TYPE =
    sequence[[state, contest]]
```
I don't see a need for these. I'd think just the county_contest_table was sufficient.
At some point for cross-county contests, we'll need to rely on the contest name being uniform across the counties.